pybind11/__main__.py

```diff
@@ -25,6 +26,7 @@ def print_includes() -> None:
 
 def main() -> None:
     parser = argparse.ArgumentParser()
+    parser.add_argument("--version", action="version", version=__version__)
```

Comment:
Could you please add `help="Prints version and exits."`

(This is almost straight from the `argparse` docs for `action` `version`, but I didn't know, and had to look it up.)

Reply:
For context, I lazily didn't bother because the argparse default is to autogenerate help text for you: "show program's version number and exit"

However I'm more than happy to make this explicit. I've tweaked your suggested wording to better fit in with the style of `__main__.py` which consistently uses imperative mood, so "Prints" -> "Print".